dclab/definitions/feat_const.py

```diff
@@ -118,6 +118,13 @@
     # Fun fact: If we had decided to compute it from the convex contour,
     # then we would have close to none pixelation effects ¯\_(ツ)_/¯.
     ["volume", "Volume [µm³]"],
+    ["area_um_raw", "Area [µm²] from a raw contour"],
+    ["per_um_raw", "Perimeter [µm] from a raw contour"],
+    ["deform_raw", "Deformation from a raw contour"],
+    ["eccentr_prnc", "Eccentricity from a raw contour"],
+    ["per_ratio", "Ratio of raw and cvx perimeters (inversed convexity)"],
```

Comment:
Do you have a link to the definition of "Convexity" and "Inverse Convexity"? I would prefer a description that aligns better with porosity, e.g. "Inverse Convexity (raw to convex perimeter ratio)" (not sure if this is technically correct).

Reply:
https://deci.ai/course/object-convexity/
https://catalyst.earth/catalyst-system-files/help/concepts/focus_c/oa_featExtract_intro_geoFeat.html
or what ever other reference... 
I do not want to calculate convexity because it is a number smaller than 1 and people are used to numbers larger that 1 for the porosity. That is why I calculate inverse convexity to be allined with porosity measure.

Reply:
OK, could you please add one link (that is likely to be there for several years) as a comment above the definition?

What do you think about "Inverse Convexity (raw to convex perimeter ratio)"?